src/Alpacka.CLI/Commands/CommandInit.cs

```diff
@@ -16,11 +16,13 @@ public CommandInit()
             Name = "init";
             Description = "Create and initialize a new alpacka pack";
             
+            var argType = Argument("type",
+                "The type of instance can be 'Vanilla', 'Server' or 'MultiMC'");
+            var argName = Argument("name",
+                "The name of the pack");
             var argDirectory = Argument("[directory]",
                 "The directory to initialize the pack in. Created if necessary.");
             
-            var optName = Option("-n | --name",
-                "Sets the pack name", CommandOptionType.SingleValue);
             var optDescription = Option("-d | --description",
```

Comment:
I.. guess the description and authors should be removed. Whoever uses `init` command will edit the config file manually anyway.